### PR TITLE
Add and enable `BraveTalk` feature on Nightly

### PR DESCRIPTION
**Pull request by bsclifton**
Enable feature on NIGHTLY only which was introduced with https://github.com/brave/brave-core/pull/9605

For more info about rollout, please see https://github.com/brave/brave-together/issues/112

cc: @kjozwiak

### CODE REVIEW

File: seed/seed.json
Comment (simonhong) [inline seed/seed.json:690]: I think having `Study` suffix in name like `BraveTalkStudy` is its convention.
@goodov Am I right? :)

Comment (bsclifton) [reply seed/seed.json]: Good catch! Fixed 😄 👍

**Review by simonhong — APPROVED** [review]
++

**Review by goodov — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 Assuming once this lands we'll be able to QA it on the staging server?

**Comment by bsclifton** [comment]
@kjozwiak yup! we should be able to. Let me ask for more info in our Slack

**Pull request merged**
